scripts/travis-test.sh

```diff
@@ -8,20 +8,25 @@ set -eu
 export GOGC=400
```

Comment:
I think it might be nice if you documented the TEST_SUITE env var (its potential settings) in the comment at the top.

Reply:
(Done)